examples/step-35/step-35.cc

```diff
@@ -422,11 +435,11 @@ namespace Step35
     void run(const bool verbose = false, const unsigned int n_plots = 10);
 
   protected:
-    RunTimeParameters::MethodFormulation type;
+    RunTimeParameters::Method type;
 
     const unsigned int deg;
-    const double       dt;
-    const double       t_0, T, Re;
+    const double       dt, t_0, T;
+    const double       Re;
```

Comment:
In the parameters class every member got its own line. Would you mind expanding these too?